tests/all/debug/lldb.rs

```diff
@@ -12,7 +12,7 @@ fn lldb_with_script(args: &[&str], script: &str) -> Result<String> {
     let mut cmd = Command::new(&lldb_path);
 
     cmd.arg("--batch");
-    if cfg!(target_os = "macos") {
+    if cfg!(any(target_os = "macos", target_os = "ios")) {
```

Comment:
I think it's ok to skip iOS in this file, it's unlikely these tests will ever complete successfully on iOS (and they're disabled by default too)

Reply:
(ditto for all other `tests/all/debug/*.rs` files)